fury/actor.py

```diff
@@ -2317,8 +2317,8 @@ def have_2_dimensions(arr):
     return spq_actor
 
 
-def billboard(centers, colors=(0, 1, 0), scales=1, vs_dec=None, vs_impl=None,
-              gs_prog=None, fs_dec=None, fs_impl=None):
+def billboard(centers, colors=(0, 1, 0), scales=1, using_gs=False, vs_dec=None,
+              vs_impl=None, gs_prog=None, fs_dec=None, fs_impl=None):
```

Comment:
Given the nature of the new functionality, we should find a way to reuse the `gs_prog` param. Something like:

```python
def billboard(..., gs_prog='gs_generated', ...):
    if gs_prog.lower() == 'gs_generated':
        # Your super awesome GS Billboard implementation
    elif gs_prog is None:
        # Use the current billboard functionality
    else:
        # Use the user's custom GS program
```

This way, we can dispose of the `use_gs/using_gs` parameter.